posthog/queries/session_recordings/session_recording_list.py

```diff
@@ -304,7 +308,7 @@ def format_event_filters(self) -> EventFiltersSQL:
     def get_query(self) -> Tuple[str, Dict[str, Any]]:
```

Comment:
We can/should cache this or even just create a lookup table elsewhere (in postgres) to do these lookups really easily.